src/modules/routes/routes.js

```diff
@@ -1,5 +1,9 @@
 const express = require('express');
-const router = express.Router();
+const Router = require('express').Router
+const userController = require('../controllers/user-controller');
+const router = new Router()
+const {body} = require('express-validator')
+const authMiddleware = require('../middlewares/auth-middleware')
```

Comment:
lines 
const Router = require('express').Router  and
const router = new Router()
can be replaced:
const router = express.Router();